sql-plugin/src/main/scala/org/apache/spark/sql/rapids/mathExpressions.scala

```diff
@@ -47,8 +47,10 @@ case class GpuAcos(child: Expression) extends CudfUnaryMathExpression("ACOS") {
 case class GpuToDegrees(child: Expression) extends GpuUnaryMathExpression("DEGREES") {
 
   override def doColumnar(input: GpuColumnVector): ColumnVector = {
-    withResource(Scalar.fromDouble(180d / Math.PI)) { multiplier =>
-      input.getBase.mul(multiplier)
+    withResource(Scalar.fromDouble(180d)) { multiplier =>
+      withResource(input.getBase.mul(multiplier)) { dividend =>
+        dividend.div(Scalar.fromDouble(Math.PI))
```

Comment:
Resouce leak, use `withResource`.

Reply:
Use `withResource` to close `Scalar.fromDouble(Math.PI)`

Reply:
ok